frontend/packages/dev-console/src/components/topology/data-transforms/data-transformer.ts

```diff
@@ -56,20 +57,24 @@ export const getTrafficConnectors = (
   }, []);
 };
 
-const getBaseTopologyDataModel = (resources: TopologyDataResources): Model => {
+const getBaseTopologyDataModel = (
+  resources: TopologyDataResources,
+  monitoringAlerts: Alerts,
+): Model => {
```

Comment:
```suggestion
const getBaseTopologyDataModel = (resources: TopologyDataResources & Alerts ): Model => {
```

can we have as single argument as here we again merging it back

Reply:
added it like : `resources: { [x: string]: Alerts } | TopologyDataResources`